astronomer/providers/google/cloud/triggers/kubernetes_engine.py

```diff
@@ -0,0 +1,129 @@
+from typing import Any, AsyncIterator, Dict, Optional, Sequence, Tuple, Union
+
+from airflow.providers.cncf.kubernetes.utils.pod_manager import PodPhase
+from airflow.providers.google.cloud.operators.kubernetes_engine import (
+    GKEStartPodOperator,
+)
+from airflow.triggers.base import TriggerEvent
+from kubernetes_asyncio.client import CoreV1Api
+
+from astronomer.providers.cncf.kubernetes.hooks.kubernetes_async import (
+    KubernetesHookAsync,
+)
+from astronomer.providers.cncf.kubernetes.triggers.wait_container import (
+    WaitContainerTrigger,
+)
+
+
+class GKEStartPodTrigger(WaitContainerTrigger):
+    """
+    Fetch GKE cluster config and wait for pod to start up.
+
+    :param location: The name of the Google Kubernetes Engine zone or region in which the
+        cluster resides
+    :param cluster_name: The name of the Google Kubernetes Engine cluster the pod should be spawned in
+    :param use_internal_ip: Use the internal IP address as the endpoint.
+    :param project_id: The Google Developers Console project id
+    :param gcp_conn_id: The google cloud connection id to use
+    :param impersonation_chain: Optional service account to impersonate using short-term credentials
+    :param regional: The location param is region name
```

Comment:
the description seems unclear for the param `regional`

Reply:
Actually, I took this from the sync operator but I have improved it a bit cec166754ce9975850d793695820ad9fa3200805